service/src/main/kotlin/fi/espoo/evaka/document/archival/ArchiveChildDocumentService.kt

```diff
@@ -32,15 +55,193 @@ class ArchiveChildDocumentService(
+                        //
+                        // .asXMLGregorianCalendar()
+                        //                                            eventType =
+                        // archiveProcessHistoryEntry.state.name
+                        //                                            eventTxId =
+                        // "rowIndex_${archiveProcessHistoryEntry.rowIndex}"
+                        //                                            eventAgent =
+                        // HistoryEventType.EventAgent().apply {
+                        //                                                any = JAXBElement(
+                        //                                                    QName("", "text"),
+                        //                                                    String::class.java,
+                        //
+                        // archiveProcessHistoryEntry.enteredBy.name
+                        //                                                )
+                        //                                            }
+                        //                                        }
+                        //                                    } ?: emptyList()
+                        //                                )
+                        //                            }
+
+                        // Security and retention policies
+                        policies =
+                            StandardMetadataType.Policies().apply {
+                                disclosurePolicy =
+                                    DisclosurePolicyType().apply {
+                                        disclosureLevel =
+                                            if (documentMetadata.confidential == true)
+                                                DisclosureLevelType.CONFIDENTIAL
+                                            else DisclosureLevelType.PUBLIC
+                                    }
+                                // TODO Evaka_Särmä_metatietomääritykset.xlsx sanotaan, että nämä
+                                // tulisi olla "Ei turvallisuusluokiteltu". Tarkista että
+                                // InformationSecurityLevelType.UNCLASSIFIED on OK
+                                informationSecurityPolicy =
+                                    InformationSecurityPolicyType().apply {
+                                        securityLevel = InformationSecurityLevelType.UNCLASSIFIED
+                                    }
+                                // TODO tarvitaanko näitä XML puolelle?
+                                // Evaka_Särmä_metatietomääritykset.xlsx sanotaan, että nämä tulisi
+                                // tulla PUT sanoman headereissa
+                                retentionPolicy =
+                                    RetentionPolicyType().apply {
+                                        retentionPeriod = "100"
```

Comment:
tää pitäis tulla `ArchivedProcess.archiveDurationMonths` vaikka käytännössä onkin aina 100v nykyään